scripts/release_info.py

```diff
@@ -17,11 +17,14 @@
 """Script that simplifies releases by collecting various information.
 Should be run from the oppia root dir.
 """
+import argparse
 import collections
 import os
 import re
 import subprocess
 
+import github
```

Comment:
@nithusha21, does the path to PyGitHub need to be inserted into sys.path for this to work?